safari/Shared (Extension)/Resources/_locales/en/messages.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extension_name": {
+        "message": "Shadowmap Location",
+        "description": "Visualize sunlight and shadow anywhere on Earth"
```

Comment:
Shadowmap Location allows you to open the current location from Google Maps/Earth in Shadowmap and vice versa.